scripts/getStatus.js

```diff
@@ -11,6 +11,8 @@ var exec = require('child_process').exec,
 
 var swaggersToProcess = utils.swaggers;
 var readmesToProcess = utils.readmes;
+//readmesToProcess = readmesToProcess.slice(0, 10);
+//swaggersToProcess = swaggersToProcess.slice(0, 10);
```

Comment:
Can you remove these commented lines?

Reply:
Removed...they were in the original script for debugging but not needed.